server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/VacationResponseGetMethodContract.scala

```diff
@@ -482,14 +482,16 @@ trait VacationResponseGetMethodContract {
       .asString
       .stripMargin
 
-    assertThatJson(response).isEqualTo(
+    assertThatJson(response)
+      .whenIgnoringPaths("methodResponses[0][1].description")
```

Comment:
Why ignoring the description path here, and then why keeping it and changing the description in the assert below then?

Reply:
Honestly I gave up trying to understand the escape sequence after 1 hour +.

(a Json in a JSON in a scala string).

I see little value in spending my time doing it.

If you wonna give it a shot though, contribs are welcome.

Reply:
It just feels pointless of having ignoring description and then a description in the assert... Maybe better to drop the description field in the assert json answer then, and having potentially a comment explaining quickly why, if somebody wants to give it a shot later?